scripts/notify-lark.js

```
@@ -0,0 +1,117 @@
+                        ],
+                        triggers.length && [
+                            { "tag": "text", "text": `Triggers: ` },
+                            { "tag": "text", "text": triggers.join(', ') }
+                        ],
+                    ].filter(Boolean)
+                }
+            }
+        }
+    };
+
+    const res = await Axios.post(chatURL, body, { headers });
```

ESLint

[error] Delete `··`

(prettier/prettier)